modules/post/windows/gather/credentials/foxmail.rb

```diff
@@ -84,14 +83,13 @@ def enum_session_file(fpath)
       while index < file.length
         if (file[index] && file[index] > "\x20" && file[index] < "\x7f" && file[index] != "\x3d")
           buffer += file[index]
-          if ['Email', 'IncomingServer', 'OutgoingServer', 'Password'].include?(buffer)
-            email_info[buffer] = find_string(file, index + offset) || nil
-          elsif ['IncomingPort', 'OutgoingPort'].include?(buffer)
-            email_info[buffer] = find_string(file, index + 5, 2) || nil
-          elsif ['InComingSSL', 'OutgoingSSL'].include?(buffer)
-            email_info[buffer] = find_string(file, index + 5, 2) == 1 || false
-          else
-            pass
+          case buffer
+            when 'Email', 'IncomingServer', 'OutgoingServer', 'Password'
+              email_info[buffer] = find_string(file, index + offset) || nil
+            when 'IncomingPort', 'OutgoingPort'
+              email_info[buffer] = find_string(file, index + 5, 2) || nil
+            when 'InComingSSL', 'OutgoingSSL'
+              email_info[buffer] = find_string(file, index + 5, 2) == 1 || false
```

Comment:
Should there not be a default case here for if no match is found?

Reply:
Yes, but do nothing by default.

Reply:
Ah fair point yeah this code will just continue if a match isn't found. My bad, current code is perfectly fine, going to mark this as resolved.